workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/MonoBehaviourActivationManager.cs

```diff
@@ -88,8 +97,8 @@ public void EnableSpatialOSBehaviours()
         {
             foreach (var behaviour in behavioursToEnable)
             {
-                var componentIdsToReaderWriterLists = injector.InjectAllReadersWriters(behaviour, entity);
-                store.AddReaderWritersForBehaviour(behaviour, componentIdsToReaderWriterLists);
+                var injectablesFromIds = injector.InjectAllRequiredFields(behaviour, entity);
```

Comment:
How about renaming this to `injectableIdToInjectables` or `injectedInjectables`?